packages/medusa/src/subscribers/order.js

```diff
@@ -54,6 +59,21 @@ class OrderSubscriber {
       })
     )
   }
+
+  updateDraftOrder = async data => {
+    const order = await this.orderService_.retrieve(data.id)
+
+    const draftOrder = await this.draftOrderService_.retrieveByCartId(
+      order.cart_id
+    )
+
+    if (draftOrder) {
+      await this.draftOrderService_.registerCartCompletion(
+        draftOrder.id,
+        order.id
+      )
+    }
```

Comment:
Possibly more relevant within a `createFromDraft`

Reply:
Could also be incorporated in above suggestion.